presto-main/src/main/java/com/facebook/presto/sql/analyzer/MaterializedViewPlanValidator.java

```diff
@@ -74,9 +78,24 @@ protected Void visitLogicalBinaryExpression(LogicalBinaryExpression node, Materi
         if (!node.getOperator().equals(LogicalBinaryExpression.Operator.AND)) {
             throw new SemanticException(NOT_SUPPORTED, node, "Only AND operator is supported for join criteria for materialized view.");
         }
+
```

Comment:
remove this line

Reply:
Hi @gracexin2003 , it looks great to me and approving. Can you help address James' comment here and remove this redundant empty line?